Packs/Code42/Scripts/Code42GetDepartingEmployeeAlerts/Code42GetDepartingEmployeeAlerts.py

```diff
@@ -0,0 +1,36 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+def main():
+    res = {"total": 0}
+    res_data = []
+    top = demisto.args().get("top") or 10
+
+    try:
+        employees = demisto.executeCommand("code42-departingemployee-get-all", {})[0]["Contents"]
+        res["total"] = len(employees)
```

Comment:
If we are ignoring certain employees later, should we not set `total` after the for loop?

Reply:
Depending on what the count represents, if it represents all employees irrespective of whether they have alerts or not then this is ok.